### PR TITLE
#4146 - Minor change to text in contact module

### PR DESCRIPTION
**Pull request by stpaultim**

### CODE REVIEW

**Review by olafgrabienski — COMMENTED** [review]
Maybe use the `!url` placeholder not only in the link but also in the text?

Current:
> available by default at /contact

Proposal:
> available at !url

**Comment by stpaultim** [comment]
@olafgrabienski - Great idea. DONE!

File: core/modules/contact/contact.admin.inc [hunk 1]
```diff
@@ -166,7 +166,7 @@ function contact_form_permissions() {
 function contact_category_list($form, $form_state) {
   $form['help'] = array(
     '#type' => 'markup',
-    '#markup' => t('Configure the categories shown in the <a href="!url">site-wide contact form</a> here. All categories are shown in a select list on the contact form, which will send to different addresses when selected.', array('!url' => url('contact'))),
+    '#markup' => t('Configure the categories shown in the site-wide contact form available at <a href="!url">!url</a>. All categories are shown in a select list on the contact form, which will send to different addresses when selected.', array('!url' => url('contact'))),
```
Comment (klonos) [inline core/modules/contact/contact.admin.inc:169]: RTBC 👍

**Review by klonos — APPROVED** [review]
RTBC 👍

**Comment by quicksketch** [comment]
Merged into 1.x and 1.16.x.

**Pull request merged**